ranzen/torch/utils.py

```diff
@@ -90,3 +91,38 @@ def __exit__(self, *args: Any) -> None:
 
     def __repr__(self) -> str:
         return f"Event of duration: {self.time}"
+
+
+def batchwise_pdist(x: Tensor, chunk_size: int = 1000, p_norm: float = 2.0) -> Tensor:
+    """Compute pair-wise distance in batches.
+
+    This is sometimes necessary because if you compute pdist directly, it doesn't fit into memory.
+
+    :param x: Tensor of shape (N, F) where F is the number of features.
+    :param chunk_size: Size of the chunks that are used to compute the pair-wise distance. Larger
+        chunk size is probably faster, but may not fit into memory.
```

Comment:
Shouldn't there be a line-break here as the description for chunk_size spans multiple lines?

Reply:
Pretty sure sphinx can parse this just fine.

Reply:
![image](https://user-images.githubusercontent.com/7741417/174157310-aece187c-9a6d-4de6-8051-381b88ce0080.png)